Test/Solver/Proof.expected

```diff
@@ -0,0 +1,23 @@
+query:
+(set-logic QF_UF)
+(assert false)
+(check-sat)
+(get-proof)
+
+res: unsat
+
+proof:
+import Smt.Reconstruction.Certifying.Boolean
+import Smt.Reconstruction.Certifying.Resolution
+import Smt.Reconstruction.Certifying.Factor
+import Smt.Reconstruction.Certifying.PermutateOr
+
+set_option maxRecDepth 10000
+set_option maxHeartbeats 500000
+
+
+theorem th0 : False → False :=
```

Comment:
The Lean syntax here is not correct, the function has no body.

Reply:
We also need to work on the cvc5 printer 😅. Long term, we agree with you that proof lifting should be done in Lean, but we will continue to do the lifting in cvc5 for the short term.